examples/meals.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type Meal struct {
+	Title   string
+	Comment string
+}
+
+func main() {
+	var meals = []Meal{
+		{Title: "Bread", Comment: "Good, not so healthy"},
+		{Title: "Eggs", Comment: "Healthy"},
+		{Title: "Apple", Comment: ""},
+		{Title: "Burger", Comment: "Really?"},
+	}
+
+	answers := struct {
+		Meal string
+	}{}
+
+	titles := make([]string, len(meals))
+	for i, m := range meals {
+		titles[i] = m.Title
+	}
+	var qs = []*survey.Question{
+		{
+			Name: "meal",
+			Prompt: &survey.Select{
+				Message: "Choose a meal:",
+				Options: titles,
+				Description: func(value string, index int) string {
+					return meals[index].Comment
+				},
+			},
+			Validate: survey.Required,
+		},
+	}
+
+	// ask the question
+	err := survey.Ask(qs, &answers)
```

Comment:
Could this use AskOne since there is only one question being asked? I think it would result in simpler code.

Additionally, the destination for the answer could be a pointer to an int. That way, survey will write the index of the selected feature upon completion. I think the index would be more useful for the user, since they can query the original `meals` slice with it.